src/components/Modals/FiatRamps/views/RampsList.tsx

```diff
@@ -1,17 +1,88 @@
 import { ChevronRightIcon } from '@chakra-ui/icons'
 import { Box, Button, Flex, Stack, Tag } from '@chakra-ui/react'
+import { useMemo } from 'react'
+import { ReactElement } from 'react'
 import { useHistory } from 'react-router-dom'
-import gemlogo from 'assets/gem-mark.png'
-import onjunologo from 'assets/onjuno.png'
 import { AssetIcon } from 'components/AssetIcon'
 import { Card } from 'components/Card/Card'
 import { Text } from 'components/Text'
 
+import { FiatRamp, SupportedFiatRampConfig, supportedFiatRamps } from '../config'
 import { FiatRampsRoutes } from '../FiatRampsCommon'
 
-export const RampsList = () => {
+type RampsListProps = {
+  setFiatRampProvider: (fiatRamp: FiatRamp) => void
+}
+
+export const RampsList: React.FC<RampsListProps> = ({ setFiatRampProvider }) => {
   const history = useHistory()
 
+  const ramps = useMemo(() => {
+    type Entry = [keyof typeof supportedFiatRamps, SupportedFiatRampConfig]
+    const initial: ReactElement[] = []
+    const result = (Object.entries(supportedFiatRamps) as Entry[]).reduce((acc, entry) => {
+      const [fiatRamp, fiatRampConfig] = entry
+      if (fiatRampConfig.isImplemented) {
+        acc.unshift(
+          <Button
+            key={fiatRamp}
+            width='full'
+            height='auto'
+            justifyContent='space-between'
+            variant='ghost'
+            fontWeight='normal'
+            py={2}
+            onClick={() => {
+              setFiatRampProvider(fiatRamp)
+              history.push(FiatRampsRoutes.Manager)
+            }}
+            rightIcon={<ChevronRightIcon boxSize={6} />}
+          >
+            <Flex flex={1} flexDirection='row' justifyContent='space-between' alignItems='center'>
+              <Flex flexDirection='row' justifyContent='center' alignItems='center'>
+                <AssetIcon src={fiatRampConfig.logo} />
+                <Box textAlign='left' ml={2}>
+                  <Text fontWeight='bold' translation={fiatRampConfig.label} />
+                  <Text translation={fiatRampConfig.info ?? ''} />
+                </Box>
+              </Flex>
+              <Box>
+                <Tag colorScheme='green' mr={2}>
+                  <Text translation='fiatRamps.buy' style={{ textTransform: 'uppercase' }} />
+                </Tag>
+                <Tag colorScheme='gray'>
+                  <Text translation='fiatRamps.sell' style={{ textTransform: 'uppercase' }} />
+                </Tag>
+              </Box>
+            </Flex>
+          </Button>,
+        )
+      } else {
+        acc.push(
```

Comment:
I don't like the idea of showing unimplemented features like this. 

I'd prefer that we use feature flags to turn on and off incomplete features.

Reply:
@cjthompson this was a product decision for marketing reasons